qp_qiime2/tests/test_qiime2.py

```diff
@@ -463,11 +463,13 @@ def test_filter_samples(self):
         # only 1 element
         self.assertEqual(len(ainfo), 1)
         # and that element [0] should have this file
-        exp = [(join(out_dir,
-                     'filter_samples/filter_samples/feature-table.biom'),
-                'biom')]
+        exp = [(join(out_dir, 'filter_samples/filtered.biom'), 'biom')]
         self.assertEqual(ainfo[0].files, exp)
 
+        b = load_table(exp[0][0])
+        self.assertIn('taxonomy', b.metadata(b.ids(axis='observation')[0],
```

Comment:
I think it will be worth checking that the taxonomy information of 1 or 2 taxa hasn't changed and that is actually parsed ok (as an array).